pandas/core/window/numba_.py

```diff
@@ -0,0 +1,106 @@
+import types
+from typing import Any, Callable, Dict, Optional, Tuple
+
+import numpy as np
+
+from pandas._typing import Scalar
+from pandas.compat._optional import import_optional_dependency
+
+
+def make_rolling_apply(
+    func: Callable[..., Scalar],
+    args: Tuple,
+    nogil: bool,
+    parallel: bool,
+    nopython: bool,
+):
+    numba = import_optional_dependency("numba")
```

Comment:
can you add a doc-string that says what this function does (the parameters are already documented elsewhere, maybe just mention that)